__tests__/app.test.js

```diff
@@ -61,6 +60,92 @@ describe("App", () => {
           });
         });
     });
+    it("200 - We can sort by another column (author, title, ...)", () => {
+      return request(app)
+        .get("/api/articles")
+        .query({ sort_by: "title" })
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(12);
+          expect(articles).toBeSortedBy("title", { descending: true });
+        });
+    });
+    it("200 - We can change the sorting order", () => {
+      return request(app)
+        .get("/api/articles")
+        .query({ sort_by: "article_id", order: "asc" })
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(12);
+          expect(articles).toBeSortedBy("article_id", { descending: false });
+        });
+    });
+    it("200 - We can filter by an specific topic", () => {
+      return request(app)
+        .get("/api/articles")
+        .query({ sort_by: "article_id", order: "asc", topic: "mitch" })
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(11);
+          expect(articles).toBeSortedBy("article_id", { descending: false });
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
```

Comment:
At this point, we know the order of the articles coming back, so i might have picked one specific article and asserted against actual values to be extra certain